### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rochoa**
Hey @IagoLast, can I pick your latest changes (removing list dataviews) from cartodb.js?

### CODE REVIEW

**Comment by IagoLast** [comment]
sure! Add this line to the news.md for traceability

* Removed unused code from cartodb.js for deprecated list-dataviews

**Comment by alonsogarciapablo** [comment]
> sure! Add this line to the news.md for traceability
> Removed unused code from cartodb.js for deprecated list-dataviews

@iago we normally don't update NEWS.md if we're only updating the dependencies.

I'm bumping deps again to include a couple of deep-insigths.js and CartoDB.js PRs that we just merged.

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "perfect-scrollbar": "git://github.com/CartoDB/perfect-scrollbar.git#master",
```
Comment (alonsogarciapablo) [inline package.json]: @rochoa remember to change the version to `4.9.91` in this file too!

Comment (rochoa) [reply package.json]: Why?

**Pull request merged**